test/run-qemu

```diff
@@ -8,28 +8,28 @@ ARCH="${ARCH-$(uname -m)}"
 QEMU_CPU="${QEMU_CPU:-max}"
 
 set_vmlinux_env() {
-    VMLINUZ=${VMLINUZ-"/lib/modules/${KVERSION}/vmlinuz"}
```

Comment:
Since some tests are failing: How about determining KVERSION using lsinitrd in case it is not set?